app/views/layouts/application.html.erb

```diff
@@ -80,11 +80,11 @@
             </svg>
             <span class="govuk-footer__licence-description">
               All content is available under the
-              <a class="govuk-footer__link" href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" rel="license">Open Government Licence v3.0</a>, except where otherwise stated
+              <a class="govuk-footer__link" href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" rel="license">Open Government Licence v3.0</a>, except where otherwise stated <!-- how to include split up sentences -->
```

Comment:
How should this footer be handled? Does the copyright symbol need to be included? Should the split sentence be included separately

Reply:
Let's not care about the footer now. It's static anyway.